deepreg/config/parser.py

```diff
@@ -49,10 +51,9 @@ def load_configs(config_path: Union[str, List[str]]) -> Dict:
         head, tail = os.path.split(config_path[0])
         filename = "updated_" + tail
         save(config=loaded_config, out_dir=head, filename=filename)
-        logging.error(
-            f"Used config is outdated. "
-            f"An updated version has been saved at "
-            f"{os.path.join(head, filename)}"
+        logger.error(
+            "Used config is outdated. An updated version has been saved at %s.",
```

Comment:
a bit outside of this ticket, but maybe "The provided config file is in an outdated format...."

Reply:
updated